### PR TITLE
Add missing Transition animations

### PR DESCRIPTION
**Pull request by mic4ael**

### CODE REVIEW

**Comment by mic4ael** [comment]
Don't Merge! I am still writing a `Dockerfile` to build it.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.PHONY: build
+
+build:
+	rm -rf node_modules
+	docker run -it --entrypoint /bin/bash --workdir /indico-sui-theme --mount type=bind,src=`pwd`,target=/indico-sui-theme node:10 build.sh
```
Comment (ThiefMaster) [inline Makefile]: run the container as your own uid, no need to build as root (and more important, no need to `npm install` all the stuff as root). also, can we avoid creating the `node_modules` folder outside the container?

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+npm i;
```
Comment (ThiefMaster) [inline build.sh]: shell scripts don't need semicolons unless you want to put more than one command in a line

File: Makefile (hunk 1 above)
Comment (ThiefMaster) [inline Makefile]: what's the benefit of using `--mount ...` instead of `-v $(pwd):/indico-sui-theme`?

File: build.sh (hunk 2 above)
Comment (mic4ael) [reply build.sh]: Oh.. I knew you would complain..

File: Makefile (hunk 1 above)
Comment (mic4ael) [reply Makefile]: It is the preferred way of creating volumes now.

File: build.sh (hunk 2 above)
Comment (mic4ael) [reply build.sh]: Yeah, I am aware of it but I always prefer to use them ;p

File: Makefile (hunk 1 above)
Comment (mic4ael) [reply Makefile]: I could probably mount a tmpfs volume at the container node_modules to prevent modification of the host directory

**Review by ThiefMaster — APPROVED** [review]

**Pull request merged**